### DISCUSSION TITLE
Nomination of @fwolter

### DISCUSSION THREAD

**Discussion by kaikreuzer** [openhab/openhab-addons#14700] [Add-on Maintainers Discussions]
Dear @openhab/add-ons-maintainers,

I would like to suggest a new member to the maintainer team and thus nominate @fwolter.
I had the pleasure to review his first contribution - the extremely huge [LCN binding](https://www.openhab.org/addons/bindings/lcn) - and I was very positively surprised by his coding skills and by how deep he already went into advanced openHAB features.
I also noticed his [extensive activity](https://github.com/openhab/openhab-addons/pulls?q=is%3Apr+is%3Aopen+commenter%3Afwolter+) on reviewing other contributions and his comments are imho always professional, helpful as well as polite. From my pov, he clearly fulfills the ideal of meritocracy here and thus I wish him being part of our team - assuming and hoping that he accepts the invitation.

So @openhab/add-ons-maintainers, please cast your vote here! 👍 / 👎

**Comment by kaikreuzer** [comment]
Looks like one of the quickest votes we have done so far - so let's welcome @fwolter as a new maintainer!

**Comment by J-N-K** [comment]
Welcome!
